docs/theme/docs/typescript.py

```diff
@@ -55,7 +55,7 @@ class TypeScriptLexer(RegexLexer):
         ],
         'tag': [
             (r'\s+', Text),
-            (",", Punctuation),
+            (r'[,\|]', Punctuation),
```

Comment:
This is an artifact of syncing the "main" lexer in the configs folder back into this file, which is used only for the theme preview.

Reply:
why do we even have this file twice? Can we not use the other one in the preview as well?